src/style.css

```diff
@@ -1 +1,102 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Avenir Book';
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Avenir Heavy';
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: 900;
+  font-style: normal;
+}
+
+h1 {
+  font-family: Avenir, sans-serif;
+  display: none;
```

Comment:
The `h1` element is set to `display: none;`, which will hide it from view. If this is not intentional, consider removing this style or using a different approach to hide the element conditionally.